packages/form-js-viewer-render/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@bpmn-io/form-js-viewer-render",
+  "version": "1.8.6",
+  "description": "Render form state - powered by bpmn.io",
+  "type": "module",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "scripts": {
+    "all": "run-s test build",
+    "build": "run-p bundle generate-types",
+    "bundle": "vite build",
+    "dev": "vite",
+    "generate-types": "tsc --allowJs --skipLibCheck --declaration --emitDeclarationOnly --outDir dist/types src/index.js && copyfiles --flat src/*.d.ts dist/types",
+    "test": "vitest",
+    "prepublishOnly": "npm run build"
+  },
+  "license": "SEE LICENSE IN LICENSE",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bpmn-io/form-js.git",
+    "directory": "packages/form-js-viewer-render"
+  },
+  "author": {
+    "name": "bpmn.io contributors",
+    "url": "https://github.com/bpmn-io"
+  },
+  "sideEffects": false,
+  "files": [
+    "dist"
+  ],
+  "devDependencies": {
+    "@preact/preset-vite": "^2.8.2",
+    "vitest": "^1.5.3"
```

Comment:
@Skaiir We can install this on the project root since we don't need them to run as a peer dependency this way we avoid users installing these packages when the user installs `@bpmn-io/form-js-viewer-render`